### DISCUSSION TITLE
Java DB from local volume

### DISCUSSION THREAD

**Discussion by gerbil** [aquasecurity/trivy-operator#1699] [Help & Support]
We got an issue with java db (400+mb) downloads from China. For main DB it's possible to use local volume and store it in local folder of trivy server (/home/scanner/.cache/trivy/db), but seems like the same is not possible for Java DB (/home/scanner/.cache/trivy/java-db)

Any suggestions how we could try to mitigate it?

We tried to skip java db update, but seems like it's trying to fetch it anyway: 
```
clientServerSkipUpdate = true
skipJavaDBUpdate = true
offlineScan = true
```

Log from java scanning job:
```
The first run cannot skip downloading Java DB
2023-12-15T12:27:16.172Z        ←[31mFATAL←[0m  image scan error: scan error: scan failed: failed analysis: analyze error: 
pipeline error: failed to analyze layer (sha256:22bf74b2421f27ab1892c376590f6c9a520f7cee7f5eb57cb00c98367edfe82f)
: post analysis error: post analysis error: Unable to initialize the Java DB: Java DB update failed:
'--skip-java-db-update' cannot be specified on the first run
```

**Comment by chen-keinan** [comment]
@gerbil I'm converting this discussion to issue #1700
